evthread_pthread.c

```diff
@@ -180,31 +183,42 @@ evthread_use_pthreads_with_flags(int flags)
 		evthread_posix_cond_wait
 	};
 
-	if (pthread_mutexattr_init(&attr_default))
-		return -1;
+	pthread_mutex_lock(&once_init_lock);
+	if (once_init == 1) {
+		pthread_mutex_unlock(&once_init_lock);
+		return 0;
```

Comment:
Don't we want separate return value when it is already initialized? I don't think that this is signifncant, just asking.

Reply:
I think leave the return value same with first init, this will more suitable for the elder programs, or it may be confused to the programer.